package.json

```diff
@@ -31,19 +32,36 @@
     "homepage": "https://github.com/arbitriume/server#readme",
     "dependencies": {
         "braumeister": "git://github.com/brewyah/braumeister.git",
+        "taproom": "git://github.com/brewyah/taproom.git",
```

Comment:
Pulls master at `npm install` time - can be replaced by the local version by running `npm link taproom`.